src/Symfony/Bundle/FrameworkBundle/Tests/Templating/TemplateNameParserTest.php

```diff
@@ -86,17 +85,15 @@ public function testParseValidNameWithNotFoundBundle()
     /**
      * @group legacy
      * @dataProvider provideAbsolutePaths
-     * @requires function Symfony\Bridge\PhpUnit\ErrorAssert::assertDeprecationsAreTriggered
+     * @expectedDeprecation Absolute template path support is deprecated since Symfony 3.1 and will be removed in 4.0.
```

Comment:
Sure, and all at once (there is no way to make things always green step by step)